src/CommentCard.jsx

```diff
@@ -1,11 +1,45 @@
-export default function CommentCard({ comment }) {
-  const { body, author, created_at } = comment;
-
-  return (
-    <div className="comment-card">
-      <p>{body}</p>
-      <p>comment by: {author}</p>
-      <p>created at: {created_at.slice(0, 10)}</p>
-    </div>
-  );
+import { useState, useEffect } from "react";
+import deleteCommentByID from "./Utils/deleteCommentByID";
+
+export default function CommentCard({ comment, commentList, setCommentList }) {
+  const [deleteComment, setDeleteComment] = useState(false);
+  const [hasDeleted, setHasDeleted] = useState(false);
+  const { body, author, created_at, comment_id } = comment;
+  const condition = author === "grumpy19";
+
+  useEffect(() => {
+    if (deleteComment === true) {
```

Comment:
If this delete is happening as a result of clicking a button, the api request could just go in the button handler rather than reply on state changing in the button handler function that would trigger a `useEffect`.